core/src/main/java/org/apache/carbondata/core/metadata/schema/datamap/Granularity.java

```diff
@@ -29,10 +29,16 @@
 public enum Granularity {
   YEAR("year_granularity"),
   MONTH("month_granularity"),
+  WEEK("week_granularity"),
   DAY("day_granularity"),
   HOUR("hour_granularity"),
+  FIVE_MINUTE("five_minute_granularity"),
+  TEN_MINUTE("ten_minute_granularity"),
```

Comment:
Can you keep them in sorted order ? easy to find and read

Reply:
sorted order means, i didnt get the actual point

Reply:
hour, 30 mins, 15, 10, 5, minute, second. That order (descending). Now it is not in descending or ascending order

Reply:
done